pyproject.toml

```diff
@@ -70,12 +70,35 @@ include = [
   "/.coveragerc",
 ]
 
-[tool.black]
-line-length=80
-extend-exclude="_vendor"
+# Ruff section
+[tool.ruff]
+lint.select = [
+    "I",      # isort: all
+    "PL",     # pylint: all
+    "S",      # flake8-bandit: all
+    "N",      # pep8-naming: all
+    "RUF100"  # ruff: find unused noqa
+]
```

Comment:
I would like to see at least E and F rulesets (the ruff default ones) enabled as well. 

Adding them leads to a bunch of `line-too-long` errors (apparently ruff is stricter than black when it comes to line length).  I wonder if we should just use the default 88 line-length since it looks like we have been fine with a lot of lines past 80... You could add E501 to ignore list for now: we can re-enable that afterwards or discuss changing the default line length next week, @lukpueh might have an opinion